### PR TITLE
Homework7: Alina Shautsova

### PR DESCRIPTION
**Pull request by AlinaShautsova**

### CODE REVIEW

File: homework7_statues.py
Comment (Stml89) [inline homework7_statues.py:1]: Very nice!

File: homework7_cows_and_bulls.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+while True:
+    common_cows.clear()
+    common_bulls.clear()
+    str_from_player = input("Enter you prediction: ")
+
+    list_from_player = []
+    for char in str_from_player:
+        list_from_player.append(int(char))
+
+    for item in list_from_player:
```
Comment (Stml89) [inline homework7_cows_and_bulls.py]: it will not count cows correctly

File: homework7_cows_and_bulls.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    random_element = random.choice([1, 2, 3, 4, 5, 6, 7, 8, 9, 0])
+    if random_element not in list_with_random_numbers:
+        list_with_random_numbers.append(random_element)
+
+common_cows: List[int] = []
+common_bulls: List[int] = []
+while True:
+    common_cows.clear()
+    common_bulls.clear()
+    str_from_player = input("Enter you prediction: ")
```
Comment (Stml89) [inline homework7_cows_and_bulls.py:19]: user can enter non unique number, need to handle this case

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework7_cows_and_bulls.py (hunk 1 above)
Comment (AlinaShautsova) [reply homework7_cows_and_bulls.py]: Thank you! Fixed it

File: homework7_cows_and_bulls.py (hunk 2 above)
Comment (AlinaShautsova) [reply homework7_cows_and_bulls.py:19]: Fixed it

**Review by AlinaShautsova — COMMENTED** [review]
Fixed everything

**Review by Stml89 — APPROVED** [review]

**Pull request merged**